templates/definition/meter/deye-hybrid-3p.yaml

```diff
@@ -110,6 +110,82 @@ render: |
       address: 588 # "battery capacity"
       type: holding
       decode: uint16
+  batterymode:
+    source: switch
+    switch:
+    - case: 1 # normal
+      set:
+        source: sequence
+        set: 
+        - source: const
+          value: 25 # SoC TOU 1 100%
+          set: 
+            source: modbus
+            {{- include "modbus" . | indent 10 }}
+            register:
+              address: 166 # soc time of use 1
+              type: writemultiple
+              decode: int32s
+        - source: const
+          value: 0 # start time TOU Slot 1
+          set: 
+            source: modbus
+            {{- include "modbus" . | indent 10 }}
+            register:
+              address: 148 # soc time of use 1
+              type: writemultiple
+              decode: int32s
+        - source: const
+          value: 2355 # end time TOU Slot 1
+          set: 
+            source: modbus
+            {{- include "modbus" . | indent 10 }}
+            register:
+              address: 149 # soc time of use 1
+              type: writemultiple
+              decode: int32s
+    - case: 2 # hold
+      set:
+        source: sequence
+        set:
+        - source: const
+          value: 100 # SoC TOU 1 100%
```

Comment:
Dito

Reply:
Ja, deswegen auch `draft`. 

Wie charge umzusetzten wäre, müsste auch noch eruiert werden. Es scheint mir, als müssen da wieder mal Leistungswerte gesetzt werden, was ATM nicht gemacht wird.

Reply:
Kannst du mir einen Tipp geben, wie ich auf den Parameter `soc` zugreifen kann und diesen dann auf Register `166` setzen kann? 

[Hier ](https://github.com/deadrabbit87/evcc/blob/master/templates/definition/meter/kostal-plenticore.yaml)wird es ja so gemacht, aber ich werde da nicht schlau daraus.

Reply:
> Ja, deswegen auch `draft`.
> 
> Wie charge umzusetzten wäre, müsste auch noch eruiert werden. Es scheint mir, als müssen da wieder mal Leistungswerte gesetzt werden, was ATM nicht gemacht wird.

Laden der Batterie nach Ziel soc würde über Register `172` funktionieren. Allerdings muss hierzu zuerst in den Einstellungen das Laden der Batterie aus dem Netz global aktiviert werden. Das wäre Register `130`. 

Damit sich dieses aber aktivieren lässt, muss zuerst der Strom für das Netzladen und ein soc konfiguriert werden. (Register `127` und `128`) werden. Siehe auch hier in der Anleitung: 

![grafik](https://github.com/evcc-io/evcc/assets/24924801/5d07ddcc-34e7-4d4e-ae1c-643f9012d62e)

Mein Vorschlag wäre, dass evcc nur Register `172` setzt und die anderen beiden Register von Benutzer manuell gesetzt werden müssen. 

Eine andere Möglichkeit hätte ich jetzt für den Deye nicht rausgefunden.